standard.tex

```diff
@@ -98,54 +107,42 @@ \subsection{PCB Design}
 \item Don't batch assemble boards (if you are making multiple) when they arrive, because you will be less inclined to change your design if you notice small bugs if you've already committed lots of components (which are worth far more than the cheap boards) to this design. Build one, test to death, then decide whether to make another revision or to assemble the rest.
 \end{itemize}
 \end{itemize}
-%end of pcb subsection -- }}}
+%-- }}}
 
-%start of miscellaneous subsection -- {{{
+%Miscellaneous -- {{{
 \subsection{Miscellaneous}
 \begin{itemize}
-\item All switches need idiot-proof labeling. Labeling should be permanent (i.e. not tape, etc).
-\item All mission critical components should have a hotswappable replacement. You should have that replacement's location written down. "Hotswappable" means you can replace it without any tools (except maybe pliers or a screwdriver. But no generator).
+\item All switches and connectors need idiot-proof labeling. Labeling should be permanent (i.e. not on tape, etc).
+\item All mission critical components should have a hotswappable replacement. You should have that replacement's location written down. "Hotswappable" means you can replace it without any powered tools.
 \item If it's conceivably possible for a component to be installed backwards, there should be idiot proof documentation of which way it goes, that documentation should be less than 2 inches from the install point.
-\item Buy spares. You will fuck things up, be prepared.
-\item Don't solder at hotter than 700\degree. Even that is probably pretty aggressive. Use leaded solder, it melts at a low temperature so you won't burn off all your flux. If your solder isn't melting below 700\degree, the iron's tip is probably bad and should be replaced. Take care of your tips!
-\item Don't short things. If you have any bare wires flailing around, don't apply power to the system.
-\item All battery connections should be terminated in a female connector to prevent battery shorting and enable easy battery swapping.
-\item Before ordering any components, make a list of all components in your design, and go through their data sheets one by one and ensure that they meet or exceed the requirements of your design. Do not just trust the Digikey table values, they are occasionally mislabeled.
+\item Buy spares for all components provided it is not prohibitively expensive.
+\item All battery connectors should be terminated in a female connector to prevent battery shorting and enable easy battery swapping.
+\item Before ordering any components, make a list of all components in your design, and go through their data sheets one by one and ensure that they meet or exceed the requirements of your design. Do not just trust the Digi-Key table values, they are occasionally mislabeled. Design reviewers should also do this.
```

Comment:
proof of this should happen before a design review - reviewee posts a document saying what they've checked

Reply:
Added a comment in the design review section.